src/go/util/file/types.go

```diff
@@ -22,21 +24,22 @@ type ImageDetails struct {
 	Size     int
 }
 
-type ExperimentFile struct {
+type File struct {
 	Name       string   `json:"name"`
 	Path       string   `json:"path"`
 	Date       string   `json:"date"`
-	Size       int      `json:"size"`
+	Size       int64    `json:"size"`
```

Comment:
Why the switch from `int` to `int64`? Is it just because `fs.FileInfo.Size` is an `int64`?

Reply:
That's the main reason. Though it also seems like it doesn't hurt to be explicit on the int size since a 32 bit int would be problematic for files > 4.4GB.